internal/run/consumer.go

```diff
@@ -146,14 +146,16 @@ func (c *Consumer) Consume(log *zap.Logger, repository *model.Repository) (err e
 		annotations["effx.io/file-path"] = effxYAMLFile
 
 		if lang != "" {
-			tags["lang"] = lang
+			annotations["effx.io/vcs-connect-inferred-tags"] = "language"
+			tags["language"] = lang
```

Comment:
do we need to / should we lowercase lang?

Reply:
the function that generates it lower cases it. 
if you have a preference for putting that here instead I'm okay with that

Reply:
we should probably do it here just in case, we shouldn't depend on other systems to do it (in case it ever gets removed from the backend then this will start getting weird)

Reply:
yeah that's a cool call